src/Qir/Runtime/prerequisites.ps1

```diff
@@ -3,12 +3,14 @@
 
 #Requires -Version 6.0
 
+Write-Host "##[info] Runtime/prerequisites.ps1"
+
 if ($Env:ENABLE_QIRRUNTIME -ne "false") {
     if (($IsWindows) -or ((Test-Path Env:/AGENT_OS) -and ($Env:AGENT_OS.StartsWith("Win")))) {
         if (!(Get-Command clang        -ErrorAction SilentlyContinue) -or `
             !(Get-Command clang-format -ErrorAction SilentlyContinue) -or `
             (Test-Path Env:/AGENT_OS)) {
-            choco install llvm --version=13.0.0 --allow-downgrade
+            choco install llvm --version=14.0.6 --allow-downgrade
```

Comment:
The readme under this same Runtime folder should be updated to say we use version 14 too.

Reply:
I haven't been looking at that README for probably a year :-) Thanks for the reminder. Updated.